openrtb_ext/request.go

```diff
@@ -35,6 +35,13 @@ type ExtRequestPrebid struct {
 	// passing of personally identifiable information doesn't constitute a sale per CCPA law.
 	// The array may contain a single sstar ('*') entry to represent all bidders.
 	NoSale []string `json:"nosale,omitempty"`
+
+	BidReqConversions *ExtRequestCurrency `json:"currency,omitempty"`
+}
+
+type ExtRequestCurrency struct {
+	ConversionRates map[string]map[string]float64 `json:"rates"`
+	UsePBSRates     *bool                         `json:"usepbsrates"`
```

Comment:
If we don't define it as a pointer, it would always default to `false` and the requirements in #1140 estate that we should default to true. For those `bidRequest.ext` fields that don't explicitly set `"usepbsrates"` to `true`, we still want that field to default to true so we can also use PBS currency rates. In other words defining it as a pointer gives us the ability to tell whether it:
- Was explicitly listed in `bidRequest.ext` and it came with a `false` value
- Was explicitly listed in `bidRequest.ext` and it came with a `true` value
- Was not listed in `bidRequest.ext` and it we don't know what value it would've had, but we want it to default to `true`.
 
Whereas if a regular `bool` JSON field:
- Was explicitly listed in `bidRequest.ext` and it came with a `false` value
- Was explicitly listed in `bidRequest.ext` and it came with a `true` value
- Was not listed listed in `bidRequest.ext`, defaults to `false` when unmarshaled.